src/main/k/working/search/search.k

```diff
@@ -8,6 +8,8 @@ module SEARCH
       | "final1" | "final2"
       | "unreachable"
 
+  rule 123 => 456 [smt-lemma]
```

Comment:
I guess that this tests that the code does not crash and (maybe) that the SMT accepts the syntax you are sending. Could you add some meaningful tests for the new code (preferably both unit tests and integration ones)?